CHANGELOG.rst

```diff
@@ -6,6 +6,23 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+2.5.2 (2020-10-21)
```

Comment:
This should not be done manually, but with cookietemple!

qube itself origins from the answer to the question of life and everything:
`https://github.com/cookiejar/cookietemple`

It's not yet available on PyPI, because of time reasons, but just clone it -> make install -> `cookietemple bump-version`

When in doubt ask me or @sven1103 . He's done it already.